default/generated/azure/tests/data/03-azure-database-config/application.properties

```diff
@@ -0,0 +1,4 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/mysql_db
+spring.datasource.username=dbuser
+spring.datasource.password=admin123
```

Comment:
same as above

Reply:
sure, replaced all existing passwords to placeholders